.travis.yml

```diff
@@ -32,12 +32,13 @@ install:
 before_script:
   - pwd
   - wget --no-verbose http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.3.1/swagger-codegen-cli-2.3.1.jar -O swagger-codegen-cli.jar
-  - java -jar swagger-codegen-cli.jar generate -i https://raw.githubusercontent.com/ga4gh/dockstore/${WEBSERVICE_VERSION}/dockstore-webservice/src/main/resources/swagger.yaml -l typescript-angular -o src/app/shared/swagger -c swagger-config.json
+  - java -jar swagger-codegen-cli.jar generate -i https://raw.githubusercontent.com/ga4gh/dockstore/develop/dockstore-webservice/src/main/resources/swagger.yaml -l typescript-angular -o src/app/shared/swagger -c swagger-config.json
   - psql -c "create user dockstore with password 'dockstore' createdb;" -U postgres
   - psql -c "ALTER USER dockstore WITH superuser;" -U postgres
   - psql -c 'create database webservice_test with owner = dockstore;' -U postgres
   - psql -f travisci/db_dump.sql webservice_test
-  - java -jar dockstore-webservice-${WEBSERVICE_VERSION}.jar server travisci/web.yml 1>/dev/null &
+  - java -jar dockstore-webservice*.jar db migrate -i 1.5.0 travisci/web.yml
```

Comment:
For consistency in this file, just use `${WEBSERVICE_VERSION}`

Reply:
won't that try to `db migrate -i` a snapshot once in a while?

Reply:
oops, misunderstood